js/components/discoveryPanel.js

```diff
@@ -111,7 +113,7 @@ var DiscoveryPanel = React.createClass({
     var countDownClock = (
       <div className={discoveryCountDownWrapperStyle}>
         <a className="discoveryCountDownIconStyle" onClick={this.handleDiscoveryCountDownClick}>
-          <CountDownClock {...this.props} timeToShow={this.props.skinConfig.discoveryScreen.countDownTime} ref="CountDownClock" />
+          <CountDownClock {...this.props} timeToShow={this.props.skinConfig.discoveryScreen.countDownTime} clockWidth={75} ref="CountDownClock" />
           <span className={this.props.skinConfig.icons.pause.fontStyleClass}></span>
```

Comment:
What happens when these numbers get out of sync with the CSS? Should we pull them out into CONSTANTS so that we don't have magic numbers in the code?